mobile/lib/common/status_screen.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart';
+import 'package:get_10101/common/service_status_notifier.dart';
+import 'package:provider/provider.dart';
+
+class StatusScreen extends StatefulWidget {
+  const StatusScreen({required this.fromRoute, super.key});
+
+  final String fromRoute;
+
+  @override
+  State<StatusScreen> createState() => _StatusScreenState();
+}
+
+class _StatusScreenState extends State<StatusScreen> {
+  @override
+  Widget build(BuildContext context) {
+    ServiceStatusNotifier serviceStatusNotifier = context.watch<ServiceStatusNotifier>();
+
+    final orderbookStatus =
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Orderbook));
+    final coordinatorStatus =
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Coordinator));
+    final overallStatus = statusToString(serviceStatusNotifier.overall());
+
+    return Scaffold(
+      appBar: AppBar(title: const Text("Status")),
+      body: SafeArea(
+        child: Column(
+          children: [
+            const SizedBox(height: 20),
+            Text("Overall: $overallStatus"),
+            const Divider(),
+            Text("Orderbook: $orderbookStatus"),
+            const SizedBox(height: 10),
+            Text("Coordinator: $coordinatorStatus")
```

Comment:
```suggestion
            Text("LSP: $coordinatorStatus")
```

Might be better to call the "coordinator" LSP or Trade Service for the user?